frontend/shared/utils/Quillify.js

```diff
@@ -98,4 +99,7 @@ export default function() {
             q.stc.enableModals();
         }
     });
+
+    // restore original focus
+    $(focusedItem).focus();
```

Comment:
fyi; I added this, now I think quill doesn't mess with which form field is focused on anymore